### DISCUSSION TITLE
[plugin:vite:css] [postcss] Unexpected token, expected "," after shadcn installed

### DISCUSSION THREAD

**Discussion by wijayanto1320** [remix-run/remix#9985] [Q&A] [answered]
Can someone help me?

when i install shadcn ui,and open then website, it's erorr

![image](https://github.com/user-attachments/assets/2f3500fe-d776-47f3-9af2-5e4e9243f6b3)

what's the problem?

I use remix js vite

**Comment by renanvalentin** [comment, marked as answer]
Check the `tailwind.config.ts` file.

```ts
fontFamily: {
  			sans: [\n          "Inter",\n          "ui-sans-serif",\n          "system-ui",\n          "sans-serif",\n          'Apple Color Emoji"',\n          'Segoe UI Emoji"',\n          'Segoe UI Symbol"',\n          'Noto Color Emoji"',\n        ]
  		},
```

it should be:

```ts
  fontFamily: {
        sans: [
          "Inter",
          "ui-sans-serif",
          "system-ui",
          "sans-serif",
          "Apple Color Emoji",
          "Segoe UI Emoji",
          "Segoe UI Symbol",
          "Noto Color Emoji",
        ],
      },
```

**Reply by wijayanto1320** [reply]
> Check the `tailwind.config.ts` file.
> 
> ```ts
> fontFamily: {
>   			sans: [\n          "Inter",\n          "ui-sans-serif",\n          "system-ui",\n          "sans-serif",\n          'Apple Color Emoji"',\n          'Segoe UI Emoji"',\n          'Segoe UI Symbol"',\n          'Noto Color Emoji"',\n        ]
>   		},
> ```
> 
> it should be:
> 
> ```ts
>   fontFamily: {
>         sans: [
>           "Inter",
>           "ui-sans-serif",
>           "system-ui",
>           "sans-serif",
>           "Apple Color Emoji",
>           "Segoe UI Emoji",
>           "Segoe UI Symbol",
>           "Noto Color Emoji",
>         ],
>       },
> ```

thanks the problem solve

**Reply by TobiramaSan** [reply]
Mehn!! This drove me crazy for hours. Thanks

**Reply by thorvn** [reply]
Great, you saved my life

**Reply by abba-m** [reply]
You saved my life too 🫶🏻😂

**Comment by wijayanto1320** [comment]
The problem is tailwind.config.ts,use [renanvalentin](https://github.com/renanvalentin) answer to solve

**Comment by pauljones0** [comment]
Why does the badly formed fontFamily occur? This happened to me too and this fix worked. Just need to know if  PR has been made to fix this or not.

**Reply by matsuyoshi30** [reply]
I think it's a shadcn issue. I've already made a [PR](https://github.com/shadcn-ui/ui/pull/5587).

**Comment by handfuloflight** [comment]
this bug sucks lol

**Comment by bluefire2121** [comment]
Find and replace
`\n + ten blank spaces`
in tailwind.config.ts

**Comment by basantiroomie** [comment]
been getting the same error 
![image](https://github.com/user-attachments/assets/ba1596e9-32e5-4227-b6f8-174a190dc78c)
      fontFamily: {
        'inter': ['Inter', 'sans-serif'],
        'cormorant': ['Cormorant Garamond', 'serif'],
        'source-serif': ['Source Serif 4', 'serif'],
        'commissioner': ['Commissioner', 'sans-serif'],
      },
      
      i dont understand what to do, checked everything else
